ipu-plugin/pkg/utils/utils.go

```diff
@@ -161,3 +162,39 @@ func GetAccApfMacList() ([]string, error) {
 
 	return strings.Split(strings.TrimSpace(output), "\n"), nil
 }
+
+// VerifiedFilePath validates a file for potential file path traversal attacks.
+// It returns the real filepath after cleaning and evaluiating any symlinks in the path.
+// It returns error if the "fileName" is not within the "allowedDir", point to a non-privileged location or "fileName" points to a file outside of allowed dir.
+func VerifiedFilePath(fileName string, allowedDir string) (string, error) {
```

Comment:
Looks like this was taken from a different project. Please put a note, where from where this function was taken.

Reply:
Done